src/Utils/DfTools.php

```diff
@@ -735,6 +735,14 @@ public static function getProductVariations($idProduct)
             AND (sa.id_shop IN (' . implode(', ', \Shop::getContextListShopID()) . ')
             OR (sa.id_shop = 0 AND sa.id_shop_group = ' . (int) \Shop::getContextShopGroupID() . '))'
         );
+
+        // Product attribute shop reference
+        $query->select('pas.minimal_quantity AS pas_minimal_quantity');
```

Comment:
si le cambias aquí el nombre ya no tendrías que hacer el if `DfTools::cleanString($product['pas_minimal_quantity'] ? $product['pas_minimal_quantity'] : $product['minimal_quantity']);`, no?
```suggestion
        $query->select('pas.minimal_quantity AS minimal_quantity');
```

Reply:
Tienes razón, lo he probado en local y funciona, mejor así